### PR TITLE
[VPA-340] Add current period ready event in SDK

### PR DESCRIPTION
**Pull request by dipeshfrollo**
[VPA-340] Add current period ready event in SDK

[VPA-340]: https://frollo.atlassian.net/browse/VPA-340

### CODE REVIEW

**Review by MaherKSantina — APPROVED** [review]

File: Sources/Events/Events.swift [hunk 1]
```diff
@@ -26,6 +26,7 @@ public class Events {
     internal struct EventNames {
         static let test = "TEST_EVENT"
         static let transactionsUpdated = "T_UPDATED"
+        static let currentBudgetPeriodReady = "B_CURRENT_PERIOD_READ"
```
Comment (nickfrollo) [inline Sources/Events/Events.swift:29]: Can we fix up some of these typos please? "READY"

File: Sources/Budgets/Budgets.swift [hunk 2]
```diff
@@ -34,6 +34,9 @@ public class Budgets: CachedObjects, ResponseHandler {
         self.service = service
     }
     
+    /// Notification fired when silent push notification recieved when current budget period is ready
+    public static let currentBudgetPeriodreadyNotification = Notification.Name("FrolloSDK.budgets.currentBudgetPeriodreadyNotification")
```
Comment (nickfrollo) [inline Sources/Budgets/Budgets.swift:38]: Capital R on Ready

**Review by nickfrollo — COMMENTED** [review]
Sorry I didn't get to this earlier, can we fix the typos though please?

File: Sources/Budgets/Budgets.swift (hunk 2 above)
Comment (dipeshfrollo) [reply Sources/Budgets/Budgets.swift:38]: Thanks. Fixed in next PR.

**Pull request merged**